pkg/snapshot/filesystem.go

```diff
@@ -52,6 +52,14 @@ type ResetFileSystemError struct {
 	Message string
 }
 
+type HostPathFileSystemError struct {
+	Message string
+}
+
+func (e HostPathFileSystemError) Error() string {
```

Comment:
nit pick: let's keep the type definition and the `Error()` function implementation coupled for the `ResetFileSystemError` type. something like:
```go
type ResetFileSystemError struct {
	Message string
}

func (e ResetFileSystemError) Error() string {
	return e.Message
}

type HostPathNotFoundError struct {
	Message string
}

func (e HostPathNotFoundError) Error() string {
	return e.Message
}
```